### PR TITLE
OBPIH-5967 Can edit completed putaway when go back to edit page after complete

### PR DESCRIPTION
**Pull request by drodzewicz**
Follow the discussion [in the ticket](https://pihemr.atlassian.net/browse/OBPIH-5967?focusedCommentId=149178)

### CODE REVIEW

File: grails-app/controllers/org/pih/warehouse/api/PutawayApiController.groovy [hunk 1]
```diff
@@ -52,6 +52,7 @@ class PutawayApiController {
             putawayItem.inventoryLevel = InventoryLevel.findByProductAndInventory(putawayItem.product, putaway.origin.inventory)
             putawayItem.quantityAvailable = productAvailabilityService.getQuantityOnHandInBinLocation(putawayItem.inventoryItem, putawayItem.currentLocation)
         }
+        response.setHeader("Cache-Control", "no-store")
```
Comment (alannadolny) [inline grails-app/controllers/org/pih/warehouse/api/PutawayApiController.groovy:55]: why was this needed?

Comment (alannadolny) [reply grails-app/controllers/org/pih/warehouse/api/PutawayApiController.groovy:55]: Oh yea, you cced Artur twice :smile: 
image
Now, I got it, thanks

**Review by alannadolny — APPROVED** [review]

File: grails-app/controllers/org/pih/warehouse/api/PutawayApiController.groovy (hunk 1 above)
Comment (drodzewicz) [reply grails-app/controllers/org/pih/warehouse/api/PutawayApiController.groovy:55]: :see_no_evil:

Comment (awalkowiak) [reply grails-app/controllers/org/pih/warehouse/api/PutawayApiController.groovy:55]: I am still baffled by the need for this

Comment (drodzewicz) [reply grails-app/controllers/org/pih/warehouse/api/PutawayApiController.groovy:55]: I feel like it is something that we need to discuss and decide.
I wanted to show the issue and a potential solution.

If we feel like disabling the cache in the header is not the way to go, just a single validation should be enough to prevent users mess with old data

Comment (awalkowiak) [reply grails-app/controllers/org/pih/warehouse/api/PutawayApiController.groovy:55]: Yeah, I feel like, having the validation should be enough. However, you can showcase or explain the cache store  tomorrow morning

**Review by awalkowiak — APPROVED** [review]
